getting_started/intel_debug/introduction.rst

```diff
@@ -182,7 +182,7 @@ reconfigured with tools but requires expert knowledge of the ALSA/ASoC/topology
      - CONFIG = topology variant needed for detected hardware configuration
 
 Important notices:
- - For compatibility reasons for **Meteor Lake and newer** ``/lib/firmware/intel/sof-ace-tplg`` must be symlinked to ``/lib/firmware/intel/sof-ipc4-tplg``
+ - For compatibility reasons with respect to **Meteor Lake** ``/lib/firmware/intel/sof-ace-tplg`` must be symlinked to ``/lib/firmware/intel/sof-ipc4-tplg``
```

Comment:
that change is only true if there is a kernel change to change the default path for LNL and directly use sof-ipc4-tplg.

Reply:
True, I can convert this to draft for now until we have consensus on it?
Since LNL is not released I would try to align with this change, just to make our legacy shorter.

Reply:
What I meant is that IF you want this change for LNL you need to send a kernel PR... I don't personally feel it's necessary but I am not going to lay on the tracks for a path replacement.

And once all the topology and kernel patches are merged we can merge this document and publish it.